### ISSUE TITLE
Missing validation of list type

### ISSUE THREAD

**Issue by spharion88** [gentics/mesh#713]
## Gentics Mesh Version, operating system, or hardware.
- v0.28.3 2019-01-09T16:47:15Z

## Problem
Sending any invalid type to a list field (at least one with listType number) produces 500 InternalServerError as response. 

### Schema
```json
{
  "name": "company_ids",
  "label": "Company Ids",
  "type": "list",
  "required": false,
  "listType": "number"
}
```
### Data
Some example cases that will all lead to an InternalServerError

```json
{
 "fields": {"company_ids": ""}
}
```
```json
{
 "fields": {"company_ids": 124523523}
}
```
```json
{
 "fields": {"company_ids": {"0": 1}}
}
```
```json
{
 "fields": {"company_ids": ["test"]}
}
```

Sending a number value to a string field will for example lead to the following 400 response (which I would expect for invalid lists too):
```json
{
  "i18nParameters": [
  ],
  "properties": {
  },
  "type": "generic_error",
  "message": "The field value for {teaser} is not a text value. The value was {122}"
}
```

**Comment by philippguertler** [comment]
I agree. We need better validation of posted JSON to Gentics Mesh in general. We could generate JSON schemas from the POJOs and use https://github.com/everit-org/json-schema to validate the posted JSON. That would give the user much better error messages.
